File: Code Chef/JavaScript/300-399/330 Bidding AUCTION.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    const intArray = strArray.map((strItem) => strItem.split(" ").map(Number));
    return intArray;
};

const whoBidsHigh = (bidOfAlice, bidOfBob, bidOfCharlie) => {
    ((bidOfAlice > bidOfBob) && (bidOfAlice > bidOfCharlie))
        ? console.log("Alice")
        : ((bidOfBob > bidOfAlice) && (bidOfBob > bidOfCharlie))
            ? console.log("Bob")
            : console.log("Charlie");
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n").slice(1));
    
    testcases.forEach((testcase) => {
        const [bidOfAlice, bidOfBob, bidOfCharlie] = testcase;
        whoBidsHigh(bidOfAlice, bidOfBob, bidOfCharlie);
    });
});

File: Code Chef/C Programming/300-399/330 Bidding AUCTION.c
#include <stdio.h>

int main() 
{
    int t,i,a,b,c;
    scanf("%d",&t);
    for(i=0;i<t;i++)
    {
        scanf("%d%d%d",&a,&b,&c);
        if((a>b)&&(a>c))
        {
            printf("Alice\n");
        }
        else if((b>a)&&(b>c))
        {
            printf("Bob\n");
        }
        else if((c>a)&&(c>b))
        {
            printf("Charlie\n");
        }
    }
	return 0;
}